sdk/keyvault/azure-security-keyvault-common/vcpkg/vcpkg.json

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+#
+{
+  "name": "azure-security-keyvault-common-cpp",
+  "version-string": "@AZ_LIBRARY_VERSION@",
```

Comment:
This is now a JSON string, so wrapped in quotes, where previously it was just:
`Version: @AZ_LIBRARY_VERSION@`